### ISSUE TITLE
Latest Linux build errors

### ISSUE THREAD

**Issue by eggrobin** [mockingbirdnest/Principia#670]
@Norgg is looking at them, we'll want to investigate what we can too.
http://norgg.org/files/principia_build_errors_2015050622-Borel-1210-g5e15465.html.

**Comment by eggrobin** [comment]
```
./physics/oblate_body_body.hpp:37:7: error: member initializer 'RotatingBody' does not name a non-static data member or base class
    : RotatingBody(massive_body_parameters, rotating_body_parameters),
      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
./physics/oblate_body_body.hpp:40:43: error: use of undeclared identifier 'angular_velocity'
          Vector<double, Frame>(Normalize(angular_velocity()).coordinates())) {
                                          ^
./physics/oblate_body_body.hpp:42:50: error: use of undeclared identifier 'gravitational_parameter'
    parameters_.j2_over_μ_ = *parameters_.j2_ / gravitational_parameter();
                                                ^
./physics/oblate_body_body.hpp:45:50: error: use of undeclared identifier 'gravitational_parameter'
    parameters_.j2_ = *parameters_.j2_over_μ_ * gravitational_parameter();
                                                ^
```
Those should be preceded by `this->` since they come from a templated base class.
```
./physics/forkable.hpp:97:23: error: declaration of 'Tr4jectory' shadows template parameter
    template<typename Tr4jectory>
                      ^
./physics/forkable.hpp:34:19: note: template parameter is declared here
template<typename Tr4jectory>
                  ^
```
Why is this identifier still around?

**Comment by eggrobin** [comment]
@Norgg has done some changes, here's the latest error batch http://norgg.org/files/principia_build_errors_2015081515-Bourbaki-399-g8ce4c34.html
corresponding to https://github.com/Norgg/Principia.

**Comment by pleroy** [comment]
@Norgg: I fixed quite a few Clang problems in #675.  You might want to merge/reset to get these changes and see how it works.  Sorry for not doing that earlier, but Clang 3.6.2 was unusable with VS 2015.

**Comment by pleroy** [comment]
Closing as @Norgg was able to compile Borel.  (We may have added more problems since then but that would be a different issue.)
